.travis.yml

```diff
@@ -1,34 +1,51 @@
 language: node_js
-
-node_js:
- - "8"
- - "10"
-
-addons:
-  code_climate:
-    repo_token: 6dc47f6e0d95f643d41d7e15941b0c26c87214b4362c6aa01cf1e465eaeae0d3
+sudo: false
 
 branches:
   only:
     - master
 
-sudo: false
-
 services:
-- mongodb
+  - mongodb
+
+install:
+  - npm install
 
 addons:
+  code_climate:
+    repo_token: 6dc47f6e0d95f643d41d7e15941b0c26c87214b4362c6aa01cf1e465eaeae0d3
```

Comment:
What's is code_climate and this token?

Reply:
> Quality by [Code Climate](https://codeclimate.com/) provides automated code review for test coverage, maintainability and more so that you can save time and merge with confidence.

 It was in the travis file before restructuring.  Not sure if it is current.

Reply:
Ok. I didn't realize it was already there.... NTC